### ISSUE TITLE
Sprint 5 Review

### ISSUE THREAD

**Issue by ksheedy** [the-spice-mice/web-app#65]

**Comment by ksheedy** [comment]
This week everything went fairly smoothly. I was able to accomplish all of my tasks for the most part and I only had a few issues to resolve with my classmates at the end of the sprint. Adam gave me some super helpful feedback on the semantics of my patterns that I will implement in the upcoming sprint. 

The main issue that I had this week was that I needed some not-yet-completed patterns to complete the patterns I was scheduled to complete. We tried to plan out our sprints to minimize this issue, but apparently I didn't catch every instance of this. I will add those patterns in the upcoming sprint.

**Comment by hollixholton** [comment]
This week was okay for me. I got a few things done but not everything I wanted to accomplish. I feel like I need to put a little bit more time into coding collaboratively/in class so I don't get confused when I'm trying to code on my own. (thanks KZJ and Adam for helping to troubleshoot)

I am however pleased at how far along our team is at this time. It's very nice to be ahead of the game a little bit.

**Comment by arobillard** [comment]
This week went well. I was able to create the patterns I was assigned to a standard I was happy with. I also worked with my peers providing some feedback on their patterns and helped them with debugging.

I ran into a few issues with my Jekyll implementation. I ended up being able to find a work around and the updated Patternbot fixed the problem. There were a few issues with people not making specific enough CSS, which caused overriding styles. We discussed it as a group and it should be resolved soon.
